storage.js

```diff
@@ -0,0 +1,24 @@
+const name = document.querySelector('#name');
+const email = document.querySelector('#email');
+const message = document.querySelector('#text');
+
+function storage(name, email, message) {
+  const saver = JSON.stringify({ name, email, message });
+  localStorage.setItem('data', saver);
+}
+
+function retrieve() {
```

Comment:
Only the email and message are being saved in the local storage. All the 3 field values should be saved in the local storage. Kindly fix this.
![image](https://user-images.githubusercontent.com/33268927/175474666-6ba3fde9-66da-4c09-9944-6f8187553fe5.png)